TestCases/sliding_interface/uniform_flow/uniform_SM.cfg

```diff
@@ -0,0 +1,267 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Thanks for adding so many test cases. Do you have the critical ones also added to the regression tests in Travis CI? It will help protect against things breaking in the future.

Also, are there any general options/comments that you can add to the config_template.cfg file for the sliding mesh capability? We try to keep that up to date the best we can.